test/unit/specs/store/delegation.spec.js

```diff
@@ -246,18 +247,36 @@ describe(`Module: Delegations`, () => {
     ).toBeUndefined()
   })
 
-  it(`should query delegated atoms on reconnection`, () => {
-    state.loading = true
-    const dispatch = jest.fn()
-    actions.reconnected({ state, dispatch })
-    expect(dispatch).toHaveBeenCalledWith(`getBondedDelegates`)
-  })
+  describe(`queries the delegated atoms on reconnection`, () => {
+    it(`when the user has logged in and is not loading`, async () => {
+      const dispatch = jest.fn()
+      await actions.reconnected({
+        state: { loading: true },
+        dispatch,
+        rootState: { user: { signedIn: true } }
+      })
+      expect(dispatch).toHaveBeenCalledWith(`getBondedDelegates`)
+    })
 
-  it(`should not query delegated atoms on reconnection if not stuck in loading`, () => {
-    state.loading = false
-    const dispatch = jest.fn()
-    actions.reconnected({ state, dispatch })
-    expect(dispatch).not.toHaveBeenCalledWith(`getBondedDelegates`)
+    it(`fails if delegations are loading`, async () => {
+      const dispatch = jest.fn()
+      await actions.reconnected({
+        state: { loading: false },
```

Comment:
this should be true according to the test title

Reply:
addressed